### ISSUE TITLE
code

### ISSUE THREAD

**Issue by duchaomin** [benchen4395/esspr_alibaba.com#1]
All data, code implements may be released to public through [official repository](https://github.com/alibaba), as well as the adaptive testing results for modern dual-stream models like CLIP and Align.

where? thx......

**Comment by BeHappyForMe** [comment]
大佬可以分享一下code吗~~

**Comment by benchen4395** [comment]
I'm so sorry for the too late reply. The testing data mentioned in the paper will be synchronized to the repository this week; The model training code is written using Alibaba's internal TensorFlow framework, and we will provide a config file for reference.
The disclosure of code on Alibaba requires strict company review, and after the organizational changes of the group last year, the application form was left unresolved; so sorry ^*^

**Comment by benchen4395** [comment]
> All data, code implements may be released to public through [official repository](https://github.com/alibaba), as well as the adaptive testing results for modern dual-stream models like CLIP and Align.
> 
> where? thx......

-----------------------------------------------------------------------------------------
-- Now we release the training and eval datasets for public research.
- [Training dataset](https://huggingface.co/datasets/benchen4395/esspr_data4train): All (a total of 31 million item pairs) are the same with those for the experiments of esspr.
- [Eval dataset](https://huggingface.co/datasets/benchen4395/esspr_data4eval): a total 5196 manually annotated pairs covering 15 industries and 973 sub-categories for the offline testing.

The data format is formulated as: 'cate_lv1_name,cate_name,trigger_id,tirgger_title,recall_id,recall_title,trigger_img,recall_img,label'

Both the image and title for each item are online grapped, with only id_value is virtual, in order to protect privacy.

-- __If you use this dataset, please cite this article. Thanks.__
